<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebsiteLockerWidget</class>
 <widget class="QWidget" name="WebsiteLockerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>خيارات</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>اسم المستخدم</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="userNameComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>168</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="blockSiteGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>168</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="colapseButton">
            <property name="toolTip">
             <string>إخفاء كل المواقع</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="CoEIAHajeb.qrc">
              <normaloff>:/images/colapse.png</normaloff>:/images/colapse.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="expandButton">
            <property name="toolTip">
             <string>إظهار كل المواقع</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="CoEIAHajeb.qrc">
              <normaloff>:/images/expand.png</normaloff>:/images/expand.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>أدوات</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QPushButton" name="addWebsiteButton">
          <property name="toolTip">
           <string>إضافة موقع جديد</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>إضافة موقع      </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/addWeb.png</normaloff>:/images/addWeb.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="addCategoryButton">
          <property name="toolTip">
           <string>إضافة تصنيف جديد</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>إضافة تصنيف   </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/addCat.png</normaloff>:/images/addCat.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="deleteButton">
          <property name="toolTip">
           <string>حذف عنصر من القائمة</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>حذف عنصر      </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/deleteItem.png</normaloff>:/images/deleteItem.png</iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="resetButton">
          <property name="toolTip">
           <string>تصفير القائمة واستعادة القائمة الإفتراضية</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>استعادة القائمة</string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="mergeButton">
          <property name="toolTip">
           <string>دمج القائمة الحالية مع القائمة الإفتراضية</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>دمج القائمة     </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/merge.png</normaloff>:/images/merge.png</iconset>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="updateButton">
          <property name="toolTip">
           <string>تحديث القائمة عبر الإنترنت</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>تحديث القائمة  </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>96</width>
            <height>202</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="saveButton">
          <property name="toolTip">
           <string>حفظ التغييرات</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>حفظ التغييرات </string>
          </property>
          <property name="icon">
           <iconset resource="CoEIAHajeb.qrc">
            <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="blackListRadioButton">
          <property name="text">
           <string>القائمة السوداء</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="whiteListRadioButton">
          <property name="text">
           <string>القائمة البيضاء</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="CoEIAHajeb.qrc"/>
 </resources>
 <connections/>
</ui>
